experimental/traffic-portal/src/app/core/core.module.ts

```diff
@@ -126,10 +130,12 @@ export const ROUTES: Routes = [
 		RegionsTableComponent,
 		RegionDetailComponent,
 		CacheGroupDetailsComponent,
-		CoordinatesTableComponent,
-		CoordinateDetailComponent,
+		StatusesTableComponent,
+		StatusDetailsComponent,
 		TypesTableComponent,
 		TypeDetailComponent,
+		CoordinatesTableComponent,
+		CoordinateDetailComponent,
```

Comment:
Coordinates components don't need to be moved, do they?

Reply:
that was auto merge. if we have any order we can follow it.